terminus/models/lines_and_arcs_builder.py

```diff
@@ -86,25 +85,25 @@ def index_nodes(self, nodes):
                     previous_segment = LineSegment(previous_element_point, previous_segment_new_end_point)
 
                     d2 = previous_segment_new_end_point.squared_distance_to(next_segment_new_start_point)
-                    cos = math.cos(angle_between_vectors)
+                    cos = math.cos(math.radians(angle_between_vectors))
                     radius = math.sqrt(d2 / (2 * (1 - cos)))
 
                     # Keep the angle in the [-180, 180) range
-                    if angle_between_vectors_in_degrees >= 180:
-                        angle_between_vectors_in_degrees = angle_between_vectors_in_degrees - 360
+                    if angle_between_vectors >= 180:
```

Comment:
That's a good point, I didn't dare to touch more than correcting the conversion between radians and degrees, but you are right, I'll fix this.